### PR TITLE
Material branch

### PR DESCRIPTION
**Pull request by paullewis**
R @gauntface

### CODE REVIEW

File: src/jekyll/_plugins/custom_tags/audio.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    def render(context)
+      # If the audio source is empty then this tag shouldn't be used
+      raise "Missing source for audio tag" if @audioUrl.empty?
+
+      # Expand this page variable to be its actual value.
+      if not context[@audioUrl].nil?
+        @audioUrl = context[@audioUrl]
+      end
+
+      out = '<audio class="embedded-audio" src="' + @audioUrl +  '" controls></audio>'
```
Comment (petele) [inline src/jekyll/_plugins/custom_tags/audio.rb]: @paullewis - can you check how much Chrome will preload here and potentially add the preload=none attribute if it's more than a few kb?

File: src/jekyll/_plugins/custom_tags/audio.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Copyright 2014 Google Inc. All rights reserved.
```
Comment (petele) [inline src/jekyll/_plugins/custom_tags/audio.rb]: Is there a reason why you did this as an audio tag vs an include? Personally I prefer includes, but not set absolutely.

File: src/jekyll/_plugins/custom_tags/ytvideo.rb [hunk 3]
```diff
@@ -45,6 +45,7 @@ def render(context)
       # If the video source is empty then this tag shouldn't be used
       raise "Missing source for video tag" if @videoId.empty?
 
+      # Expand this page variable to be its actual value.
```
Comment (petele) [inline src/jekyll/_plugins/custom_tags/ytvideo.rb]: If this is a TODO, please file an issue and reference it here.

File: src/jekyll/_plugins/custom_tags/audio.rb (hunk 1 above)
Comment (paullewis) [inline src/jekyll/_plugins/custom_tags/audio.rb]: Oh I'll just add preload=none. Would rather we did everything on demand.

File: src/jekyll/_plugins/custom_tags/audio.rb (hunk 2 above)
Comment (paullewis) [inline src/jekyll/_plugins/custom_tags/audio.rb]: Just to be in-keeping with the ytvideo embed code, and @gauntface said it was his preference, though I guess I can leave you two to fight that out? :runner:

File: src/jekyll/_layouts/shows/show.liquid [hunk 4]
```diff
@@ -24,10 +24,18 @@
       Previous Seasons &amp; Episodes
     </h6>
     {% for subdirectory in page.context.subdirectories reversed %}
+      {% comment %}
+        TODO(gauntface): find better solution for podcasts in shows.
```
Comment (petele) [inline src/jekyll/_layouts/shows/show.liquid]: Please file an issue for this and reference it in the code

File: src/jekyll/_plugins/custom_tags/audio.rb (hunk 2 above)
Comment (petele) [inline src/jekyll/_plugins/custom_tags/audio.rb]: :facepunch: - nah, that's too much work. if it's his, then we'll go with that.

Comment (gauntface) [inline src/jekyll/_plugins/custom_tags/audio.rb]: *fist bump*

Generally the YouTube got out of hand and an include is to easy for someone
to jump in an alter, also include may be missing something and will still
print nothing, tags give easier error checking and throwing.

**Comment by petele** [comment]
LGTM - merging.

**Pull request merged**